services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -52,3 +58,40 @@ func handleUnsupportedMethod(w http.ResponseWriter, r *http.Request) {
 		klog.Errorf("failed write data to response writer: %v", err)
 	}
 }
+
+func getAllocatedQuotaInGib(r *http.Request) (*int, error) {
+	var quota = struct {
+		StorageQuota int    `json:"storageQuota"`
+		Unit         string `json:"unit"`
+	}{}
```

Comment:
Do we really need a custom struct? can't we use the known quantity struct to decode into?

Reply:
Yes we need a custom quantity to decode.